app/src/test/java/org/schabi/newpipe/settings/ContentSettingsManagerTest.kt

```diff
@@ -166,6 +166,18 @@ class ContentSettingsManagerTest {
         assertFalse(contains)
     }
 
+    @Test
+    fun `IsValid must return false if the zip has not the two necessary files`() {
+        val newpipe = File(classloader.getResource("settings/newpipe.zip")?.file!!)
```

Comment:
Use `File(classloader.getResource("settings/newpipe.zip")!!.file` like in the other tests ( `!!` up front and no `?`).

This removes the redundant null check, which is done because of the `?`. It is redundant because with the `!!` afterwards it will immediatly throw an exception if it is null.

Reply:
Ah okay.  In that case, all other calls in the other tests should be changed (all other change, except one) uses that structure. Should I do it or you want to do it yourself later?

Reply:
Please do. I must have missed it when adding it myself 😄